Dockerfile.server.base

```diff
@@ -0,0 +1,19 @@
+FROM node:9-stretch
+
+ARG CI
+ENV CI=$CI
+ENV PROJECT_TYPE="server"
+
+VOLUME [ "/opt/orbs/logs", "/opt/orbs/db" ]
+
+# TODO: move to later stage, use cache for ts compiler & top level software
+
+ADD package.json yarn.lock .yarnrc docker/bootstrap-*.sh docker/install*.sh /opt/orbs/
```

Comment:
You don't need to include `bootstrap-sdk.sh` or `install*.sh` here (the later is unneeded at all)